rllib/offline/estimators/test_ope.py

```diff
@@ -0,0 +1,151 @@
+import unittest
```

Comment:
Nice, thanks for adding unit tests to this entire complex!

Reply:
Actually, can we:
* move this into a new sub-dir `esimators/tests`
* add this to our BUILD file right above:

```
<- HERE
py_test(
    name = "offline/evaluators/tests/test_ope",
    tags = ["team:ml", "offline"],
    size = "small",
    srcs = ["offline/evaluators/tests/test_ope.py"]
)
<- END: HERE

# --------------------------------------------------------------------
# Policies
# rllib/policy/
#
# Tag: policy
# --------------------------------------------------------------------

py_test(
    name = "policy/tests/test_compute_log_likelihoods",
    tags = ["team:ml", "policy"],
    size = "medium",
    srcs = ["policy/tests/test_compute_log_likelihoods.py"]
)

```